test-files/cmd/glacial/serial/replacement-test-archive/ServiceBuildingReplacementTest.sh.off

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+set -e
+
+# Wholesale Building Replacement Workflow
+#
+# This test is lenient regarding the AOI, in that it may modify some features in the ref data that lie just outside of the AOI. Secondary 
+# features crossing the AOI may be included in the output or conflated with reference features. This workflow should work other polygon 
+# data types, but only buildings have been tested with it so far.
+
+TEST_NAME=ServiceBuildingReplacementTest
+IN_DIR=test-files/cmd/glacial/serial/$TEST_NAME
```

Comment:
Tests in the replacement-test-archive folder are the first versions of the tests that called multiple commands. I want to keep a version of them in history for reference, then will remove them.